components/wso2is7.key.manager/src/main/java/org/wso2/is7/client/WSO2IS7KeyManager.java

```diff
@@ -689,6 +691,13 @@ public void loadConfiguration(KeyManagerConfiguration configuration) throws APIM
                             (APIConstants.KeyManager.KEY_MANAGER_OPERATIONS_USERINFO_ENDPOINT);
         }
 
+        String schemasEndpoint;
+        if (StringUtils.lowerCase(userInfoEndpoint).contains("/me")) {
+            schemasEndpoint = userInfoEndpoint.replaceAll("(?i)/me$", "/Schemas");
+        } else {
+            throw new APIManagementException("Error occurred while loading the key manager configuration");
```

Comment:
Shall we be a bit descriptive here @dakshina99 ?
```suggestion
            throw new APIManagementException("Error occurred while getting the schemas endpoint from the user info endpoint. The user info endpoint doesn't contain the string \"/me\"");
```